paasta_tools/check_marathon_services_replication.py

```diff
@@ -130,5 +130,4 @@ def check_service_replication(
         instance_type_class=marathon_tools.MarathonServiceConfig,
         check_service_replication=check_service_replication,
         namespace=None,  # not relevant for mesos
-        mesos=True,
```

Comment:
We're probably able to delete this file entirely (though it probably makes sense to do it as a follow-up.)